modules/multiple_evaluator.py

```diff
@@ -0,0 +1,38 @@
+"""
+Module to evaluate multiple Evaluator objects to determine if drone should RTL
+"""
+
+from . import evaluate_condition_abstract
+
+
+class MultipleEvaluator():
+    """
+    Attributes: 
+        evaluate_object_list : A list of EvaluateCondition objects, representing conditions 
+        to evaluate when determining RTL
```

Comment:
Nit: Remove when RTL. This evaluator framework can be generalized so it can be used to evaluate conditions for any case. For example, we can also use this evaluator framework to evaluate if the drone should change altitude or change heading.